src/show-dialog-confirmation.ts

```diff
@@ -0,0 +1,24 @@
+import { fireEvent } from "custom-card-helpers";
+
+export interface ConfirmationDialogParams {
+  title?: string;
+  text: string;
+  cancel?: string;
+  okay?: string;
+  confirm?: () => void;
+  dismiss?: () => void;
+}
+
+export const loadConfirmationDialog = () =>
+  import(/* webpackChunkName: "confirmation" */ "./dialog-confirmation");
+
+export const showConfirmationDialog = (
+  element: HTMLElement,
+  systemLogDetailParams: ConfirmationDialogParams
+): void => {
+  fireEvent(element, "show-dialog", {
+    dialogTag: "dialog-confirmation",
+    dialogImport: loadConfirmationDialog,
```

Comment:
@bramkragten  This here made me think I couldn't just fire it, but had to have a reference to it as well

Reply:
Oh yeah, you are right... open a confirm dialog from HA first when `dialog-confirmation` is not registered as custom component and then fire your own? If it is imported already it won't need the `dialogImport`...